docs/pages/introduction/why-not-expo.mdx

```diff
@@ -4,107 +4,49 @@ sidebar_title: Limitations
-
-Apps built with `eas build -p ios|android` [include only your app's explicit native dependencies](https://blog.expo.dev/expo-managed-workflow-in-2021-d1c9b68aa10), and so this limitation does not apply if you use EAS Build. [Learn about how to use it](/build/introduction).
-
-</Collapsible>
-
-> We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
-
-## Limitations of the bare workflow
-
-In the bare workflow, we have full access to the underlying native projects and any native code. It's a "bare" native project with React Native and one or more packages from the Expo SDK installed. Anything that you can do in a native project is possible here.
-
-The following list is therefore specifically oriented towards the limitations that exist around using Expo tools and services in the bare workflow.
-
-<Collapsible summary={<span><span className="strike">Build service only works in the managed workflow</span> (✅ Resolved in December 2020)</span>}>
-
-~~To build your app binaries for distribution on the Apple App Store and Google Play Store you will need to follow the same steps that you would in any native project, the Expo build service can't handle it for you. We are working on bringing bare workflow support to the build service in the near future.~~
-
-You can now use [EAS Build](/build/introduction) to build and sign your apps just as easily as in the managed workflow! [Read the announcement blog post](https://blog.expo.dev/expo-application-services-eas-build-and-submit-fc1d1476aa2e).
-
-</Collapsible>
-
-<Collapsible summary="Configuration must be done on each native project rather than once with app.json">
-
-Configuring app icons, launch screen, and so on must be configured in the native projects for each platform using the standard native tooling, rather than once using a simple JSON object.
-
-</Collapsible>
+**We strongly recommend any projects that require additional libraries with native code to migrate to [development builds](/development/introduction/). It's like creating a version of Expo Go that is specifically customized to your app's needs.**
 
 ## Up next
 
-If you've been reading along each section of the introduction then you will have a pretty good high-level understanding of Expo tools.
+We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or [GitHub discussions](https://github.com/expo/expo/discussions).
```

Comment:
```suggestion
```
I'm struggling to suggest some alternative wording here. I'm uncomfortable saying these limitations are in our roadmap to resolve, because several of them are either impossible in the current mobile ecosystem (e.g., Expo Go limitations), or involve tradeoffs we're unlikely to accept (e.g., some edge cases in EAS Build).

The wording of "if anything is missing" is tricky, too, because the context above is limitations, but those links are for features. If we're missing a limitation on this page, it probably is actually best to post in forums or edit the doc in a PR, which are features built into the bottom of each page already.

Thus, I was wondering, what if we strike this paragraph altogether?

Reply:
I agree and too find this wording irrelevant with the current set of tools and features.